packages/indexer/src/handlers/events/substrateChains/handleRequestEvent.handler.ts

```diff
@@ -32,6 +33,11 @@ export async function handleSubstrateRequestEvent(event: SubstrateEvent): Promis
 		return
 	}
 
+	if (!isHyperbridge(sourceId) && !isHyperbridge(hostId)) {
```

Comment:
Do this in the if block for skipping aggregated requests
```suggestion
	if (!isSubstrateChain(sourceId) || (!isHyperbridge(sourceId) && isHyperbridge(hostId))) {
```